packages/kotti-ui/source/make-props.ts

```diff
@@ -140,6 +140,7 @@ const walkSchemaTypes = <SCHEMA extends z.ZodTypeAny>(
 			)
 		}
 
+		case z.ZodFirstPartyTypeKind.ZodAny: // special case for "no type check" props
```

Comment:
Is this still necessary? If yes, this should have tests.

Reply:
test without 🤷🏼